### DISCUSSION TITLE
Connect to multiple cloud

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [Sid72020123/scratchconnect#9] [Q&A] [answered]
Can i connect to multiple cloud projects to make a network of cloud projects? (scratchnet xD)

**Comment by Quantum-Codes** [comment]
Try for yourself

**Reply by Quantum-Codes** [reply]
Bruh. I have exams... who asked you? @Sid72020123 pls say xDDD

**Reply by Sid72020123** [reply]
Do you mean connecting two projects with scratchconnect and setting the cloud variables?

**Reply by Quantum-Codes** [reply]
Yes. Like 1 python repl can change 2 project's cloud

```python
user.connect_project(19281929) #too lazy to see syntax :P
user.connect_project(191919) #connects to both? Or disconnects from 1st
```
If disconnects then how to connect to both project?

**Reply by Sid72020123** [reply]
scratchconnect is object oriented so you can connect any number of projects you want!
Test this code:
```python
import scratchconnect

user = scratchconnect.ScratchConnect("Username", "Password")

project1 = user.connect_project(project_id=1)
project2 = user.connect_project(project_id=2)

variables1 = project1.connect_cloud_variables()
variables2 = project2.connect_cloud_variables()

# Program to set cloud variables:
set1 = variables1.set_cloud_variable(variable_name="Name", value=123)  # Set a Cloud Variable
set2 = variables2.set_cloud_variable(variable_name="Name", value=123)  # Set a Cloud Variable

if set and set2:
    print("Cloud Variable Updated!")
```

**Reply by Sid72020123** [reply]
Try the above code if it works! I cannot test it now but I will test it tomorrow.......
It should work!

**Reply by Sid72020123** [reply]
I tested it just now and it worked!

**Reply by Quantum-Codes** [reply]
GREAT!!!  So set 1 and 2 returns true?

**Reply by Sid72020123** [reply]
Yes! I also checked the cloud variables of those projects and it worked fine!

**Reply by Quantum-Codes** [reply]
:)

**Comment by Sid72020123** [comment, marked as answer]
See the answer here: https://github.com/Sid72020123/scratchconnect/discussions/9#discussioncomment-1016220
